worldedit-core/src/main/java/com/sk89q/worldedit/internal/Constants.java

```diff
@@ -44,19 +44,18 @@ private Constants() {
 
     /**
      * The DataVersion for Minecraft 1.13
+     * @deprecated If Fawe drops interaction with 1.13, this method is subject to removal.
```

Comment:
Can we even deprecate it for removal? Afaik it's needed for a variety of schematic shenanigans for 1.13 created schematics